nemo_curator/modules/semdedup/utils.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
```

Comment:
To keep things consistent, I'd prefer moving to `utils/semdedup` or something similar instead.

Reply:
Resolved by https://github.com/NVIDIA/NeMo-Curator/pull/130/commits/e11988035cf8711168bb1c7c6729efeae0d42650